### ISSUE TITLE
Firefox plugins don't work anymore

### ISSUE THREAD

**Issue by petercommand** [NixOS/nixpkgs#76943]
**Describe the bug**

A recent commit to firefox (https://hg.mozilla.org/integration/autoland/rev/9b4e603eb86a ) removed support for loading extensions with the environment variable MOZ_PLUGIN_PATH, and the firefox wrapper in nixpkgs uses this environment variable to load plugins.

**To Reproduce**
1. install firefox
2. set nixpkgs.config.firefox.enableAdobeFlash = true
3. execute firefox

**Expected behavior**

The flash plugin should load

**Screenshots**

**Additional context**

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.89, NixOS, 19.09.1625.0dc46b0e1c8 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-19.09.1625.0dc46b0e1c8, nixos-unstable-20.03pre205710.352f030b715"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```
@edolstra 
@andir

**Comment by taku0** [comment]
Related issue of upstream: https://bugzilla.mozilla.org/show_bug.cgi?id=1602308

**Comment by peti** [comment]
The solution "wait-for-upstream" does not seem to work well. Upstream does not seem to care.

**Comment by bjornfor** [comment]
There is activity now in the upstream issue, and a patch has been sent for review (direct link: https://phabricator.services.mozilla.com/D68601). I really hope this gets fixed soon.

**Comment by bjornfor** [comment]
And it landed!

**Comment by ajs124** [comment]
@bjornfor Did it make it into a release? Has this been resolved?

**Comment by bjornfor** [comment]
@ajs124: Hi, thanks for reminder about this issue. I just checked that setting `nixpkgs.config.firefox.enableAdobeFlash = true;` enables adobe flash, so this is fixed! (I'm on release-20.03 branch, commit 985047d3c9fb16bb05bc8cde7d2a1a703f670eae, which has firefox 80.0.)
